pmm-app/src/shared/components/Elements/Icons/Database.tsx

```diff
@@ -0,0 +1,55 @@
+/* eslint max-len: 0 */
+
+import React, { FC } from 'react';
+import { useTheme } from '@grafana/ui';
+import { getStyles } from './Icons.styles';
+
+export const Database: FC = (props) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <svg
+      width="31"
+      height="38"
+      viewBox="0 0 31 38"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
```

Comment:
Consider whether we need passing props here.

Reply:
We need it to be able to pass additional parameters from the outside, we're not using it here but use for some other icons, after your comment I understood that it would be better if we refactor it so I'll create a separate task

Reply:
Sorry, I will have to either decline this PR or fix myself.

Reply:
Fixed!